README.md

```diff
@@ -11,9 +11,8 @@ Our default implementation works with Amazon Web Services (AWS), but AWS Amplify
 *Note: Amplify docs are still being updated and will go live by EOW. The below links will take you to the SDK documentation currently.
```

Comment:
Docs are going live today. So it might be misleading to say that they will go live EOW.

Reply:
removed note